app/helpers/shopping/attribute_accessible_helper.rb

```diff
@@ -0,0 +1,8 @@
+module Shopping
+  module AttributeAccessibleHelper
+    def method_missing(name, *args, &block)
```

Comment:
This module allows us to call out to attr_accesible in the shopping repo without having the gem as a dependency. Box still uses attr_accessible, so this keeps shopping from erroring out w/o the dependency and allows us to set the accessible attributes in the gem rather than reopening the model classes in box